DataFormats/HLTReco/src/classes_def.xml

```diff
@@ -21,7 +21,6 @@
   <class name="std::vector<HLTPerformanceInfo::Module>::const_iterator"/>
 
   <class name="HLTPerformanceInfo::Path" ClassVersion="11">
-   <version ClassVersion="10" checksum="3100415018"/>
```

Comment:
You can't remove a class version as it has been used. You can only add (or remove those which you
know have not been used, as those intermediate ones below)